test/rails_utils_test.rb

```diff
@@ -202,7 +202,7 @@
       let(:action_name) { "update" }
 
       it "does not generate an additional javascript method" do
-        view.javascript_initialization.wont_match "Dummy.#{controller_name}.init_();"
+        view.javascript_initialization.wont_match "Dummy.#{controller_name}..init();"
```

Comment:
Hi Juanito, it is not a typo. I believe this test is testing if there is no
custom js method provided, it should not generate an empty method name.
On Thu, 17 Mar 2016 at 10:37 PM Juanito Fatas <notifications@github.com>
wrote:

> In test/rails_utils_test.rb
> <https://github.com/winston/rails_utils/pull/26#discussion_r56513879>:
>
> > @@ -202,7 +202,7 @@
> >        let(:action_name) { "update" }
> >
> >        it "does not generate an additional javascript method" do
> > -        view.javascript_initialization.wont_match "Dummy.#{controller_name}.init_();"
> > +        view.javascript_initialization.wont_match "Dummy.#{controller_name}..init();"
>
> ..init();" typo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/winston/rails_utils/pull/26/files/7bc9b02dd2a7e64ade28eeba39156c9dee89c76d#r56513879>
>